src/Products/PloneMeeting/migrations/migrate_to_4217_1.py

```
@@ -0,0 +1,49 @@
+                    row['css_class'] = css_class
+                    row['action'] = 'remove'
+                    row['replace_new_content'] = ''
+                    row['replace_new_css_class'] = ''
+                    row['powerobservers'] = cfg.hideCssClassesTo
+                    res.append(row)
+            cfg.setCssTransforms(res)
+            delattr(cfg, 'cssClassesToHide')
+            delattr(cfg, 'hideCssClassesTo')
+        logger.info('Done.')
+
+    def run(self, extra_omitted=[], from_migration_to_4200=False):
```

Ruff (0.15.7)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)